src/utils/clients.ts

```diff
@@ -111,7 +111,6 @@ const setDeviceState = (() => {
 		} else {
 			deviceStates[uuid].targetState = state;
 		}
-		deviceStates[uuid].targetState = state;
```

Comment:
I think you need to remove the “else” block instead

Reply:
Never mind, confused.

Reply:
Nope because in the `if` block we create the object with `{..., targetState: state }` so it's only in the existing object case (the else) that we need to update it.  It doesn't technically break anything to do this redundant assignment, but it is confusing and unnecessary